test/quantization/test_quantize_jit.py

```diff
@@ -2982,3 +3002,17 @@ def test_single_linear_dynamic(self):
                     qconfig_dict,
                     debug=True)
                 self.assertEqual(model_fake_quantized(self.calib_data[0][0]), result_eager)
+
+    @skipIfNoFBGEMM
+    def test_linear_dynamic_fp16(self):
+        linear_model = SingleLayerLinearModel().eval()
+        model_eager = quantize_dynamic(linear_model, dtype=torch.float16)
+        result_eager = model_eager(self.calib_data[0][0])
+        model_script = torch.jit.script(linear_model)
+        model_traced = torch.jit.trace(linear_model, self.calib_data[0][0])
+        qconfig_dict = {'' : float16_dynamic_qconfig}
+
+        for model in [model_traced, model_script]:
+            model_quantized = quantize_dynamic_jit(model, qconfig_dict, debug=False)
+            # TODO check model with debug=True matches quantized model result
```

Comment:
you can also check numerics with `checkGraphModeOp`

Reply:
Yeah, right now looks like debug option set to `True` isn't producing identical numerics. Once I fix/figure that out I'll use `checkGraphModeOp`